### ISSUE TITLE
Propose a way for users to contribute custom keys

### ISSUE THREAD

**Issue by esmoak** [adgad/kboard#2]
I think it would be helpful to have a spot where users can contribute their custom key setups in an easy manner. I've just learned how powerful this keyboard can be and I'm sure I haven't scratched the surface. 

If you agree, what would be the best way to accomplish this?

My thoughts are that the simplest method might be to create an issue titled **Custom Keys** then immediately close it so it doesn't clutter your open issues. Then link to that thread from the README. 

I'm not familiar with the github wiki process, etc., but if you like this idea I'm open to suggestions.

**Comment by CaptnSeraph** [comment]
I think a wiki would be better than an open/closed issue page.

It wouldn't hurt to have a more ELI5 instruction set for the custom key creation process and if no one objects I don't mind writing up some more user friendly instructions and collating suggested shortcuts.

**Comment by esmoak** [comment]
@theseraphim that sounds like a good idea.

Here are just a few I came up with that I think might be helpful to add to a list:

**Markdown**
* Select text then strikethrough: `/MDstrike!y,ds,iraw(~~~~),2j,p`
* Select text then bold: `/MDbold!y,ds,iraw(****),2j,p`

**BBCode**
* Embed YouTube video: `/YouTube!i([youtube][/youtube]),10j`
* Insert Quote: `/quote!i([quote][/quote]),8j`
* Insert Image: `/image!i([img][/img]),6j`

(Note: These can be modified to paste from the clipboard with the "pc" function if you have already copied what you want to insert between the tags.)

**General Usage**
* Spacebar: `/Space!iraw( )`
* Move cursor back 1: `/<---!j`
* Move cursor forward 1: `/--->!k`
* Backspace to delete a character: `/DELbackspace!d`
* Delete either selected text or everything: `/DELsel&all!ds`
* Create a hidden blank key with whatever function you want: `/!iraw( )` 
(Note: Replace !iraw( ) with whatever is wanted for hidden function.)

**Search**:
* Google search reddit posts: `/GReddit!i(site:reddit.com),iraw( )`
* DuckDuckGo search reddit posts: `/redditSRCH!i(!r),iraw( )`
* Google search youtube vids: `/GYouTube!i(site:youtube.com),iraw( )`
* DuckDuckGo search youtube: `/YouTubeSRCH!i(!yt),iraw( )`

These are just some examples that can be modified. "i" can be replaced with "iraw" and vice versa depending on needs.

If anyone knows how to make the equivalent of an **Undo key,** let me know. I've tried various things with pasting, using i($0), etc, but can't get anything to work.

**Comment by esmoak** [comment]
Looks like there is a wiki, so I'm going to close this. Feel free to add these at any time if wanted. 

Love this keyboard, thanks again for creating it.
